### ISSUE TITLE
Add Normalize as a transformation option

### ISSUE THREAD

**Issue by ParthivNaresh** [alteryx/evalml#2005]
I'd like the ability to normalize my input data as a transformation option.

https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.Normalizer.html#sklearn.preprocessing.Normalizer

**Comment by dsherry** [comment]
Yep. Note we should not do this for tree-based models.
